src/pet.js

```diff
@@ -0,0 +1,125 @@
+//magic numbers
+const MAX_FITNESS = 10;
+const MIN_HUNGER = 0;
+const HUNGRY = "I am hungry"
+const UNFIT = "I need a walk"
+const BOTH = "I am hungry AND I need a walk"
+const FINE = "I feel great!"
+const DIED = "Your pet is no longer alive :("
+const ERROR = new Error('Your pet is no longer alive :(')
```

Comment:
you already have the string `Your pet is no longer alive :(` declared in the variable above. You could use that variable here to make it more DRY, i.e.
```
const ERROR = new Error(DEAD);
```